sdk/core/azure-core/inc/azure/core/context.hpp

```diff
@@ -142,13 +195,76 @@ namespace Azure { namespace Core {
 
   public:
     /**
-     * @brief Constructs a new context with no deadline, and no value associated.
+     * @brief Constructs a new child context with no deadline, and no value associated.
      *
      */
     Context() : m_contextSharedState(std::make_shared<ContextSharedState>()) {}
```

Comment:
Is this creating a new child context, or a new root context? I think it's root, since there is no parent. Update doc?